examples/anthos-service-mesh-multicluster/vars.sh

```diff
@@ -0,0 +1,41 @@
+# Customize variables as needed
+
+# Terraform debug logging. 
+# Comment these two variables out if not running on local
+export TF_LOG="TRACE"
```

Comment:
trace is extremely verbose. would DEBUG be sufficient?
why do we need verbose logging anyway (once the tf modules are stable this should not be necessary)?

Reply:
Another testing artifact, removing